docs/spark-standalone.md

```diff
@@ -24,7 +24,7 @@ the master's web UI, which is [http://localhost:8080](http://localhost:8080) by
 
 Similarly, you can start one or more workers and connect them to the master via:
 
-    ./bin/spark-class org.apache.spark.deploy.worker.Worker spark://IP:PORT
+    ./sbin/start-slave.sh <worker#> <master-spark-URL>
```

Comment:
This change is not entirely equivalent since `start-slave.sh` runs `spark-daemon.sh`. Are you sure? CC @CodingCat maybe